### PR TITLE
Remove potential race condition from es service check

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Review by esoergel — APPROVED** [review]
Seems like a good idea just in general, even without the potential race condition

File: corehq/apps/hqadmin/service_checks.py [hunk 1]
```diff
@@ -93,7 +94,7 @@ def check_elasticsearch():
     if cluster_health != 'green':
         return ServiceStatus(False, "Cluster health at %s" % cluster_health)
 
-    doc = {'_id': 'elasticsearch-service-check',
+    doc = {'_id': 'elasticsearch-service-check-{}'.format(random_hex[:7]),
            'date': datetime.datetime.now().isoformat()}
     send_to_elasticsearch('groups', doc)
```
Comment (millerdev) [inline corehq/apps/hqadmin/service_checks.py]: Should this be in a `try` block with the delete in the `finally` block? If an exception occurred in here the delete would not happen.

**Review by millerdev — APPROVED** [review]

File: corehq/apps/hqadmin/service_checks.py (hunk 1 above)
Comment (dannyroberts) [reply corehq/apps/hqadmin/service_checks.py]: good point. I'll follow up after today's deploy (wanted to get this in quickly).

File: corehq/apps/hqadmin/service_checks.py [hunk 2]
```diff
@@ -93,7 +94,7 @@ def check_elasticsearch():
     if cluster_health != 'green':
         return ServiceStatus(False, "Cluster health at %s" % cluster_health)
 
-    doc = {'_id': 'elasticsearch-service-check',
+    doc = {'_id': 'elasticsearch-service-check-{}'.format(random_hex[:7]),
```
Comment (esoergel) [inline corehq/apps/hqadmin/service_checks.py]: `TypeError("'function' object has no attribute '__getitem__'",)`

**Pull request merged**